armsrc/lfops.c

```diff
@@ -1539,7 +1797,7 @@ void ReadPCF7931() {
         tries++;
         if (BUTTON_PRESS()) return;
     } while (num_blocks != max_blocks);
-end:
+ end:
```

Comment:
the whole PCF7931 code is kind of old, and doesn't follow the newer style. It uses goto, 
Someone with time over and has been focusing on LF really should have a look into it... ;)